src/raftstore/store/peer.rs

```diff
@@ -1561,6 +1586,11 @@ impl Peer {
         Ok(())
     }
 
+    // `read_index` returns a boolean to indicate the `read` is proposed or not.
```

Comment:
The function description begin by restating the name of the function is redundant and makes functions harder to rename. Just   // Returns a boolean ...